class SailAssertionError(Exception):
    pass
class SailException(Exception):
    pass
